### PR TITLE
UI-React migration

### PR DESCRIPTION
**Pull request by mmelko**
//skip-ci 
current progress: 
https://issues.jboss.org/browse/FUSEQE-4264

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/pages/customizations/extensions/TechExtensionsListComponent.java [hunk 1]
```diff
@@ -37,14 +39,20 @@ public boolean validate() {
         return $(Element.ROOT).is(visible);
     }
 
+    public static By extensionActionButton(String extension,String action) {
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/pages/customizations/extensions/TechExtensionsListComponent.java]: autoformat please

File: ui-tests/src/test/java/io/syndesis/qe/pages/customizations/extensions/TechExtensionsListComponent.java [hunk 2]
```diff
@@ -65,13 +73,13 @@ public boolean isExtensionPresent(String name) {
     }
 
     public SelenideElement getActionOnExtensionButton(String name, String action) {
-        SelenideElement extension = this.getExtensionItem(name);
-        assertThat(extension).isNotNull();
-        SelenideElement actionButton = extension.findAll(By.tagName("button")).filter(exactText(action)).first().shouldBe(visible);
-        assertThat(actionButton).isNotNull();
+//        SelenideElement extension = this.getExtensionItem(name);
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/pages/customizations/extensions/TechExtensionsListComponent.java]: remove if not used

File: ui-tests/src/test/java/io/syndesis/qe/pages/customizations/extensions/TechExtensionsListComponent.java [hunk 3]
```diff
@@ -65,13 +73,13 @@ public boolean isExtensionPresent(String name) {
...
+//        assertThat(extension).isNotNull();
+        SelenideElement actionButton = $(extensionActionButton(name,action)).shouldBe(visible);
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/pages/customizations/extensions/TechExtensionsListComponent.java]: `return $(extensionActionButton(name,action)).shouldBe(visible);`

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/connection/ChooseAction.java [hunk 4]
```diff
@@ -35,13 +39,15 @@ public boolean validate() {
...
             fail("Action element was not found in 30s.", e);
         }
 
-        $(Element.ROOT).$$(Element.TITLE).filterBy(Condition.text(name))
-                .shouldHaveSize(1).first().shouldBe(visible).click();
+        $(getAction(name)).$(Element.SELECT_BUTTON).click();
+    }
+
+    private By getAction(String action) {
+        return By.cssSelector(String.format(Element.ACTION_SELECTOR,action.replaceAll(" ","-").toLowerCase()));
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/connection/ChooseAction.java]: autoformat please

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/connection/actions/fragments/ConfigureAction.java [hunk 5]
```diff
@@ -14,7 +14,8 @@
 public class ConfigureAction extends SyndesisPageObject {
 
     private static final class Element {
-        public static final By ROOT = By.cssSelector("syndesis-integration-action-configure");
+        //TODO replace this when another selector will be in place
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/connection/actions/fragments/ConfigureAction.java]: again the same question - do we track these todos or we fix it when it breaks?

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/DataMapper.java [hunk 6]
```diff
@@ -320,4 +322,12 @@ public void addProperty(String name, String value, String type) {
         select.selectOption(type);
         dialog.$(Element.PRIMARY_BUTTON).shouldBe(visible).click();
     }
+
+    public void switchToDatamapperIframe() {
+        switchTo().frame($(Element.ATLASMAP_IFRAME).shouldBe(visible).toWebElement());
+    }
+
+    public void switchBack() {
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/DataMapper.java]: I do not like the name of this method

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/IntegrationFlowView.java [hunk 7]
```diff
@@ -193,6 +193,15 @@ public String getConnectionPropertiesText(SelenideElement connectionStep) {
         return connectionStep.$(Element.STEP_DETAILS).shouldBe(visible).getText();
     }
 
+    public void addDatamapperStep (String action) {
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/IntegrationFlowView.java]: autoformat, remove empty lines, add space after the method

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/IntegrationsList.java [hunk 8]
```diff
@@ -44,6 +44,12 @@ public void invokeActionOnItem(String title, ListAction action) {
         }
     }
 
+    @Override
+    public SelenideElement getItem(String title) {
+        final String cssselector = String.format(Element.INTEGRATION_SELECTOR,title.toLowerCase().replaceAll(" ","-"));
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/IntegrationsList.java]: autoformat please

File: ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java [hunk 9]
```diff
@@ -91,6 +90,11 @@
         public static final By LOGOUT_MENU = By.id("userMenuDropdownButton");
         public static final By LOGOUT_MENU_ACTION_TAG = By.tagName("a");
         public static final By LOGIN_BUTTON = By.className("btn");
+        public static final By NAVIGATION_PANEL = By.className("pf-c-nav");
+
+        public static final By navigationLink(String title) {
+            return By.cssSelector(String.format("a[data-testid=\"ui-%s\"]",title.toLowerCase()));
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: autoformat please

File: ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java [hunk 10]
```diff
@@ -283,9 +287,9 @@ public void validateCredentials() {
                     case SALESFORCE:
                         //disable salesforce validation as it is not stable and smoke tests should be stable
                         return;
-                    case TWITTER:
-                        service = "Twitter";
-                        break;
+//                    case TWITTER:
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: why did we disable twitter?

File: ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java [hunk 11]
```diff
@@ -325,14 +329,14 @@ public void validateCredentials() {
...
+            OpenShiftWaitUtils.waitFor(() -> $(Element.NAVIGATION_PANEL).exists(), 30 * 1000L);
         } catch (TimeoutException | InterruptedException e) {
             fail("Navigation panel was not found in 30s", e);
         }
 
-        SelenideElement selenideElement = $(By.className("nav-pf-vertical")).shouldBe(visible);
-        ElementsCollection allLinks = selenideElement.findAll(By.className("list-group-item-value"));
-        allLinks.find(Condition.exactText(title)).shouldBe(visible).click();
+        $(Element.NAVIGATION_PANEL).shouldBe(visible);
+        ;
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: remove ;

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java [hunk 12]
```diff
@@ -64,6 +69,32 @@ public void importExtension(DataTable extensionsData) throws Throwable {
...
+
+        File[] files = new File(defaultPath).listFiles((dir, name) -> !name.contains("original") && name.endsWith(".jar"));
+        assertThat(files).hasSize(1).doesNotContainNull();
+
+        String techExtensionUrl = defaultPath + files[0].getName();
+
+        Path techExtensionJar = Paths.get(techExtensionUrl).toAbsolutePath();
+
+        assertThat(techExtensionJar.toFile()).exists();
+        System.out.println(techExtensionJar.toAbsolutePath().toString());
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: log

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java [hunk 13]
```diff
@@ -64,6 +69,32 @@ public void importExtension(DataTable extensionsData) throws Throwable {
...
+
+      //  $(By.className("dnd-file-chooser")).shouldBe(Condition.visible).sendKeys(techExtensionJar.toAbsolutePath().toString());
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: remove commented code if not used

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java [hunk 14]
```diff
@@ -146,8 +176,9 @@ public void expectExtensionPresent(String name) throws Throwable {
     }
 
     @Then("^check that technical extension \"([^\"]*)\" is not visible$")
-    public void expectExtensionNonPresent(String name) throws Throwable {
+    public void expectExtensionNonPresent(String name) throws Throwable{
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: autoformat please

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java [hunk 15]
```diff
@@ -146,8 +176,9 @@ public void expectExtensionPresent(String name) throws Throwable {
...
         log.info("Verifying if extension {} is present", name);
+        //TODO is this necessary
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: is this TODO necessary?

File: ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/ChooseStepSteps.java [hunk 16]
```diff
@@ -28,11 +32,11 @@ public void chooseStep(String stepName) {
                 chooseStep.chooseStepByDescription("Send a message to the integration\\'s log.");
                 break;
             case "Data Mapper":
-                chooseStep.chooseStep(stepName);
-                chooseStep.waitForStepAppears(By.className("dataMapperBody"), 10000);
+                connectionsList.getItem(stepName).click();
+//                chooseStep.waitForStepAppears(By.className("dataMapperBody"), 10000);
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/add/ChooseStepSteps.java]: remove comment if not used

File: ui-tests/src/test/resources/features/customizations/extension-CRUD.feature [hunk 17]
```diff
@@ -37,40 +38,36 @@ Feature: Customization - Extensions CRUD
...
     Then check visibility of page "Extensions"
 
     When select "Update" action on "Log Message Body" technical extension
     Then check visibility of page "Import Extension"
 
     When upload extension "syndesis-extension-log-body"
     Then check visibility of details about imported extension
-
-    When click on the "Update" button
+    #TODO check if this was renamed to Update
```
Comment (mcada) [inline ui-tests/src/test/resources/features/customizations/extension-CRUD.feature]: do we track this TODO?

File: ui-tests/src/test/resources/features/customizations/extension-CRUD.feature [hunk 18]
```diff
@@ -37,40 +38,36 @@ Feature: Customization - Extensions CRUD
...
     Then check visibility of page "Extensions"
 
-    When navigate to the "Customizations" page
-    And click on the "Extensions" link
+    When navigate to the "Extensions" page
+ #   And click on the "Extensions" link
     Then check visibility of page "Extensions"
     And extension "Log Message Body" is present in list
-
+#
```
Comment (mcada) [inline ui-tests/src/test/resources/features/customizations/extension-CRUD.feature]: remove `#`, add empty line

File: ui-tests/src/test/resources/features/integrations/amqp-to-amqp.feature [hunk 19]
```diff
@@ -3,6 +3,7 @@
 @ui
 @amqp
 @amqbroker
+@react
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/amqp-to-amqp.feature]: we should probably remove `@react`

File: ui-tests/src/test/resources/features/integrations/amqp-to-amqp.feature [hunk 20]
```diff
@@ -19,16 +20,16 @@ Feature: Integration - AMQP to AMQP
...
 
     # select connection as 'start' point
     And check that position of connection to fill is "Start"
     When select the "AMQP" connection
-    And select "Subscribe for Messages" integration action
-    And fill in values
-      | Destination Name | cheese |
-      | Destination Type | Queue  |
+    And select "Subscribe for messages" integration action
+    And .*fill in values by element data-testid
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/amqp-to-amqp.feature]: remove `.*`

File: ui-tests/src/test/resources/features/integrations/amqp-to-amqp.feature [hunk 21]
```diff
@@ -38,9 +39,9 @@ Feature: Integration - AMQP to AMQP
     When select the "AMQP" connection
     And select "Publish messages" integration action
     # TODO: there's a mismatch between persistent on ActiveMQ and deliveryPersistent property on AMQP
-    And fill in values
-      | Destination Name | apple |
-      | Destination Type | Queue |
+    And .*fill in values by element data-testid
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/amqp-to-amqp.feature]: remove `.*`

File: ui-tests/src/test/resources/features/integrations/db-to-db.feature [hunk 22]
```diff
@@ -46,17 +46,19 @@ Feature: Integration - DB to DB
 
 
     # add split step
-    Then check visibility of page "Add to Integration"
+   # # Then check visibility of page "Add to Integration"
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/db-to-db.feature]: just one # or remove the line

File: ui-tests/src/test/resources/features/integrations/db-to-db.feature [hunk 23]
```diff
@@ -46,17 +46,19 @@ Feature: Integration - DB to DB
...
     When add integration step on position "0"
     And select "Split" integration step
+    And click on the "Next" button
 
     # add data mapper step
-    Then check visibility of page "Add to Integration"
+  #  # Then check visibility of page "Add to Integration"
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/db-to-db.feature]: just one # or remove the line

File: ui-tests/src/test/resources/features/integrations/db-to-db.feature [hunk 24]
```diff
@@ -264,7 +271,7 @@ Feature: Integration - DB to DB
     #@wip time_unit_id is not yet available
 
     Then fill in periodic query input with "SELECT * FROM CONTACT" value
-    And click on the "Next" button
+    And click on the "Next" buttonno mne sa zd
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/db-to-db.feature]: this does not look correct

File: ui-tests/src/test/resources/features/integrations/ftp-to-ftp.feature [hunk 25]
```diff
@@ -17,38 +17,38 @@ Feature: Integration - FTP to FTP
...
+      | exists        | Override         |
+      | tempfilename  | copyingprefix    |
+      | tempprefix    | copying_test_out |
 
     And click on the "Next" button
 
     And click on the "Done" button
 
-    Then check visibility of page "Add to Integration"
+  #  # Then check visibility of page "Add to Integration"
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/ftp-to-ftp.feature]: just onr # or remove the comment

File: ui-tests/src/test/resources/features/integrations/gmail.feature [hunk 26]
```diff
@@ -38,13 +38,13 @@ Feature: Google mail Connector
 
     When select the "My GMail Connector" connection
     And select "Send Email" integration action
-    And fill in values
-      | Email to      | jbossqa.fuse@gmail.com |
-      | Email subject | syndesis-test          |
+    And .*fill in values by element ID
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/gmail.feature]: remove `.*`

File: ui-tests/src/test/resources/features/integrations/gmail.feature [hunk 27]
```diff
@@ -38,13 +38,13 @@ Feature: Google mail Connector
...
+      | to      | jbossqa.fuse@gmail.com |
+      | subject | syndesis-test          |
     And click on the "Done" button
 
     # add split step
-    Then check visibility of page "Add to Integration"
+   # # Then check visibility of page "Add to Integration"
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/gmail.feature]: just one # or remove the comment

File: ui-tests/src/test/resources/features/integrations/service-now.feature [hunk 28]
```diff
@@ -22,55 +22,57 @@ Feature: Integration - ServiceNow-amq/log
...
     And check that position of connection to fill is "Start"
 
     When select the "ServiceNow" connection
     And select "Retrieve Record" integration action
-    And fill in values
-      | Limit of elements per page     | 1                |
-      | Query to filter the result set | number=QACUSTOM4 |
-    And select "Incident" from "table" dropdown
-    And click on the "Done" button
+    And .*fill in values by element ID
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/service-now.feature]: remove `.*`

File: ui-tests/src/test/resources/features/integrations/service-now.feature [hunk 29]
```diff
@@ -22,55 +22,57 @@ Feature: Integration - ServiceNow-amq/log
...
+    And sleep for jenkins delay or "10" seconds
     Then check that position of connection to fill is "Finish"
 
     When select the "AMQ" connection
     And select "Publish Messages" integration action
 
-    And fill in values
-      | Destination Name | incidents |
-      | Destination Type | Queue     |
+    And .*fill in values by element ID
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/service-now.feature]: remove `.*`

File: ui-tests/src/test/resources/features/integrations/service-now.feature [hunk 30]
```diff
@@ -22,55 +22,57 @@ Feature: Integration - ServiceNow-amq/log
...
+    And click on the "Create Integration" link to create a new integration.
     Then check visibility of visual integration editor
     And check that position of connection to fill is "Start"
 
     When select the "AMQ" connection
     And select "Subscribe for messages" integration action
-    And fill in values
-      | Destination Name | incidents-create |
-      | Destination Type | Queue            |
+    And .*fill in values by element ID
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/service-now.feature]: remove `.*`

File: ui-tests/src/test/resources/features/integrations/twitter-to-db.feature [hunk 31]
```diff
@@ -19,26 +19,29 @@ Feature: Integration - Twitter to Database
...
     When select the "PostgresDB" connection
     And select "Invoke SQL" integration action
-    Then fill in invoke query input with "INSERT INTO CONTACT(first_name, company) VALUES(:#first_name, :#company)" value
+    And fill in invoke query input with "INSERT INTO CONTACT(first_name, company) VALUES(:#first_name, :#company)" value
     And click on the "Done" button
 
       # add data mapper step
-    Then check visibility of page "Add to Integration"
+ #   Then check visibility of page "Add to Integration
+   #-- temporary commented -------
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/twitter-to-db.feature]: revisit this one - remove or track?

File: ui-tests/src/test/resources/features/integrations/webhook-db.feature [hunk 32]
```diff
@@ -18,20 +18,20 @@ Feature: Integration - Webhook to DB
...
-    And click on the "Create Integration" button to create a new integration.
+    And click on the "Create Integration" link to create a new integration.
     Then check visibility of visual integration editor
     And check that position of connection to fill is "Start"
 
     When select the "Webhook" connection
     And select "Incoming Webhook" integration action
-    And fill in values
-      | Webhook Token | test-webhook |
+    And .*fill in values by element data-testid
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/webhook-db.feature]: remove `.*`

File: ui-tests/src/test/resources/features/integrations/webhook-db.feature [hunk 33]
```diff
@@ -18,20 +18,20 @@ Feature: Integration - Webhook to DB
...
+      | contextpath | test-webhook |
     And click on the "Next" button
-    And fill in values
-      | Select Type | JSON Instance |
+    And .*fill in values by element data-testid
```
Comment (mcada) [inline ui-tests/src/test/resources/features/integrations/webhook-db.feature]: remove `.*`

File: ui-tests/src/test/resources/features/other/settings-oauth.feature [hunk 34]
```diff
@@ -1,8 +1,8 @@
 # @sustainer: mcada@redhat.com
-
+@react
```
Comment (mcada) [inline ui-tests/src/test/resources/features/other/settings-oauth.feature]: remove or not?

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java [hunk 35]
```diff
@@ -64,6 +69,32 @@ public void importExtension(DataTable extensionsData) throws Throwable {
         }
     }
 
+    @Given("^upload extension \"([^\"]*)\" from syndesis-extensions dir using drag and drop$")
+    public void uploadExtensionFromSyndesisExtensionsDirUsingDragAndDrop(String extensionFolderName) throws Throwable {
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: We should probably handle `Throwable`

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java (hunk 14 above)
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: We should probably handle `Throwable`

File: ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/EditorSteps.java [hunk 36]
```diff
@@ -31,15 +33,20 @@
     private IntegrationFlowView flowViewComponent = new IntegrationFlowView();
     private ChooseConnection chooseConnection = new ChooseConnection();
 
+    @And("^add data mapper step before \"([^\"]*)\" action$")
+    public void addDataMapperStepBeforeAction(String actionId) throws Throwable {
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/integrations/editor/EditorSteps.java]: We should probably handle `Throwable`

File: ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java [hunk 37]
```diff
@@ -130,13 +160,13 @@ public void uploadExtensionFromFile(String extensionFolderName) throws Throwable
 
         assertThat(techExtensionJar.toFile()).exists();
 
-        $(By.cssSelector("input[type='file']")).shouldBe(visible).uploadFile(techExtensionJar.toFile());
+        $(By.cssSelector("input")).uploadFile(techExtensionJar.toFile());
     }
 
     @When("^check visibility of details about imported extension$")
     public void importDetails() throws Throwable {
```
Comment (mcada) [inline ui-tests/src/test/java/io/syndesis/qe/steps/customizations/extensions/ExtensionSteps.java]: We should probably handle `Throwable`

File: ui-tests/src/test/java/io/syndesis/qe/fragments/common/form/Form.java [hunk 38]
```diff
@@ -46,61 +53,60 @@ public void fillBy(FillBy fillBy, Map<String, String> data) {
...
+
                 }
             } else {
                 log.warn("Input {} is not present on form!", key);
             }
         }
     }
 
+    //TODO finish filling react-codemirror
+    public void fillEditor(String data) {
```
[the remark this answers is not in the document]
Comment (mmelko) [reply ui-tests/src/test/java/io/syndesis/qe/fragments/common/form/Form.java]: Well I've just finished it :)

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/DataMapper.java (hunk 6 above)
Comment (mmelko) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/DataMapper.java]: would be switchContextBack() better ?

Comment (mcada) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/steps/DataMapper.java]: Yes and I would also add a comment why these methods exist and are necessary

File: ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/connection/actions/fragments/ConfigureAction.java (hunk 5 above)
Comment (mmelko) [reply ui-tests/src/test/java/io/syndesis/qe/pages/integrations/editor/add/connection/actions/fragments/ConfigureAction.java]: Well this was note for me. It's possible that there won't be another selector in place. Second thing is if we even need root documents validation if we have ID's to all necessary elements

File: ui-tests/src/test/resources/features/integrations/twitter-to-db.feature (hunk 31 above)
Comment (mmelko) [reply ui-tests/src/test/resources/features/integrations/twitter-to-db.feature]: just FYI temporary commented was something else.. thanks for notify

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**